roxctl/common/flags/endpoint.go

```diff
@@ -92,10 +97,13 @@ func EndpointAndPlaintextSetting() (string, bool, error) {
 	}
 
 	var usePlaintext bool
+	var defaultPort int
 	switch u.Scheme {
 	case "http":
+		defaultPort = 80
 		usePlaintext = true
 	case "https":
+		defaultPort = 443
 		usePlaintext = false
 	default:
 		return "", false, errox.InvalidArgs.Newf("invalid scheme %q in endpoint URL, the scheme should be: http(s)://<endpoint>:<port>", u.Scheme)
```

Comment:
```suggestion
		return "", false, errox.InvalidArgs.Newf("invalid scheme %q in endpoint URL, use either 'http' or 'https'", u.Scheme)
```
Since you're relaxing the requirement here (port becomes optional).